PUBLISHING-AWS.md

```diff
@@ -91,7 +91,7 @@ This isn't very discoverable yet, but it's useful for testing.
 As an example, a parameter might look like this:
 
 ```
-/your/prefix/here/aws-k8s-1.24/x86_64/1.11.0-b530f30/image_id
+/your/prefix/here/aws-k8s-1.32/x86_64/1.11.0-b530f30/image_id
```

Comment:
nit: we should update the version to a meaningful one, there is no `aws-k8s-1.32` for Bottlerocket 1.11.0.